common/scripted_triggers/wc_war_triggers.txt

```diff
@@ -9,12 +9,19 @@ is_great_invader_trigger = {
 	OR = {
 		is_existential_crisis_trigger = yes
 		is_horde_invader_trigger = yes
+		is_mogu_thunder_king_trigger = yes
 	}
 }
 is_horde_invader_trigger = {
 	any_held_title = { this = title:e_horde }
 	any_owned_story = { story_type = story_horde_invasion }
 }
+is_mogu_thunder_king_trigger = {
+	# Juke note: The Thunder King is not 100% bound to the e_pandaria_empire title, which is why it is not used in this trigger
```

Comment:
Scenario A
background: Character from another continent with an empire title conquers Pandaria and installs the Thunder King and Lei Shen now rules both empires
action: Character from another continent conquers all pandaria and usurps the `e:pandaria` title. 
consequence: Lei Shen now exclusively rules an empire on some other continent but is still considered a Thunder King crisis by the game

Scenario B
background: Character from Pandaria conquers enough of the continent and revives the Thunder King
action: another character from the same or a non hostile/evil faith conquers enough of Pandaria to Usurp `e: pandaria` but Lei Shen still holds territory on Pandaria and no other empire title
consequence: Lei Shen is no longer considered a Thunder King crisis by the game

so it is not desirable that scenario A and scenario B are both valid circumstances for Lei Shen being considered a Thunder King crisis? because it doesn't matter that his subjects consider him Thunder King, he just can't do Thunder King stuff unless he's an emperor of something? kicking him off Pandaria won't cause him to not be Thunder King anymore if he's still an emperor elsewhere?